policies/templates/gcp_always_violates.yaml

```
@@ -20,8 +20,7 @@ spec:
   crd:
     spec:
       names:
-        kind: GCPAlwaysViolatesConstraint
-        listKind: GCPAlwaysViolatesConstraintList
+        kind: GCPAlwaysViolatesConstraintv1
```

I think capitalizing V will improve readability.

```suggestion
        kind: GCPAlwaysViolatesConstraintV1
```

---

Done.